### ISSUE TITLE
`@forward` with `show` doesn't work

### ISSUE THREAD

**Issue by Fernker** [wkillerud/vscode-scss#68]
- VS Code Version: 1.84.0
- SCSS IntelliSense Version: 2.14.2
- Operating System: macOS 13.5

Reproducible Case:

Steps to Reproduce:

1.  Entry file `_entry.scss`
```
@forward './_chunk';
```
 
2. Chunk file `_chunk.scss`
```
@mixin colors() {

}
```
3.  In the consuming file:
```
@use './entry' as foo;
```

With this setup the plugin will correctly give `colors` as an intellisense option:

`color: foo.`

But if the `show` keyword is used in the entry file to hide other methods like this:

`@forward './chunk' show colors;`

It will not give the intellisense prompt after 

`color: foo.`

It will still compile correctly though since `show` with `@forward` is valid Sass module syntax.

**Comment by wkillerud** [comment]
Thanks for opening an issue. Didn't know about that feature 😄 Should be fixed in [2.14.3](https://github.com/wkillerud/vscode-scss/releases/tag/2.14.3).

**Comment by Fernker** [comment]
Thank you for getting this fixed so quickly!
